concent_api/verifier/tasks.py

```diff
@@ -100,7 +108,7 @@ def blender_verification_order(
                 subtask_id,
                 VerificationResult.ERROR.name,
                 str(exception),
-                ErrorCode.VERIFIIER_FILE_DOWNLOAD_FAILED.name
```

Comment:
@cameel comment from https://github.com/golemfactory/concent/pull/514#discussion_r195780493

> Will it crash if the file being downloaded already exists? It should.
> Any file operation where the file being created is not supposed to exist but it does should crash the verifier so that we can detect and fix the issue.

Reply:
I don't see an option to set it in unpacking function, so I would have to check manually if the any file already exists. But just a line before I am doing `clean_directory`, so there should not be any files there. Do you think we need this extra check ?

Reply:
Added, it will now crash with raising exception, is that what you expected ? Or should I catch it first and log, or handle somehow ?

Reply:
The handler on the task is supposed to catch it, report to sentry and then reraise. You do not have to do anything else.